identify/extensions.py

```diff
@@ -90,6 +91,7 @@
     'h': {'text', 'header', 'c', 'c++'},
     'hbs': {'text', 'handlebars'},
     'hcl': {'text', 'hcl'},
+    'heex': {'text', 'elixir'},
```

Comment:
these appear to have a different syntax than base elixir files -- they should probably be handled separately